content/en/docs/Performance/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Performance"
+weight: 150
+date: 2020-11-20
+description: >
+  Baker performance test cases
+---
+
+We report in this page some practical examples of Baker performances we could measure in the
+NextRoll production environment.
+
+Within NextRoll, Baker is often executed on AWS EC2 instances, and thus you find in this page
+many references to
+[EC2 instance types](https://aws.amazon.com/ec2/instance-types/) (`c5.2xlarge`, `c5.2xlarge`, etc.).
+
+### Read from S3 and write to local disk
+
+On an AWS EC2 instance of size `c5.2xlarge`, Baker can read zstandard records from S3, uncompress
+them and apply a basic filtering logic, compressing them back on local files with zstandard at
+compression level 3 and long range mode at 27 using ~90% of capacity of each vCPU (8 in total) and
+~3.5GB of RAM.  
+
+It reads and writes a total of 94 million records in 8'51" (~178k r/w records per second).
+
+On a `c5.2xlarge` instance (48vCPUs) the same test takes 2'2" (~775k r/w records per second).
+
+For this test we use 711 zstd compressed files for a total of 17 GB of compressed size and 374 GB
+of uncompressed size. The average size of each record is ~4.5 KB.
+
+### Read from S3 and write to DynamoDB (in the same region)
+
+On a `c5.4xlarge` instance, Baker reads zstd compressed files from S3 writing to DynamoDB (configured
+with 20k write capacity units) at an average speed of 60k records/s (the average size of each record
+is 4.3 KB) using less than 1 GB of memory and ~300% of the total CPU capacity (less than 20% for
+each core).
+
+The bottleneck here is the DynamoDB write capacity, so Baker can easily cope with an increased load
+just increasing the write capacity units in DynamoDB (up to 400k).
```

Comment:
repetition of increased.
repetition of dynamodb
```suggestion
The bottleneck here is the DynamoDB write capacity, so Baker could handle the additional load
caused by increasing the write capacity, up to 400k.
```
I dont understand where does that 400k come from? if at 20k write capacity we use 20% of CPU, that means that we would be maxed out at 100% CPU if we multiply the write capacity by 5, so 100k write capacity.
What am i missing?

Reply:
ok, the number is the max write capacity you can set in dynamodb, not considering our cpu. We can clarify with `just increasing the write capacity units in DynamoDB (whose max value can get up to 400k).` or similar

Reply:
Oh ok so I'm not sure It's super relevant for this page then. We can just say baker would be able to handle an additional load, without writing that number

Reply:
```suggestion
The bottleneck here is the DynamoDB write capacity, so Baker could handle the additional load caused by
a possible increase in the write capacity.
```